src/test/java/org/cyclonedx/maven/BomDependenciesTest.java

```diff
@@ -215,7 +215,7 @@ private void checkTopLevelTestComponentsAsCompile(final File projDir) throws Exc
 
     /**
      * This test ensures that any <i>compile</i> dependencies hidden by excluded types are included in the BOM if they are visible dependencies
-     * @throws Exception
+     * @throws Exception if something unexpected happens
```

Comment:
Nit: we really need to document tests?

Reply:
or there is a warning at build time: just added minimum text to avoid warning